github-orgs.yml

```diff
@@ -47,3 +47,17 @@ getsentry:
       - 'sentry-android-gradle-plugin'
       - 'sentry-dotnet'
       - 'sentry-dart-plugin'
+codecov:
+  appAuth:
+    appId: 343171
+    privateKey: 'GH_APP_PRIVATE_KEY_FOR_CODECOV'
+    installationId: 40036835
+  project:
+    nodeId: 'PVT_kwDOAH2Fnc4ARzsq'
+    fieldIds:
+      productArea: 'PVTSSF_lADOAH2Fnc4ARzsqzgLXyZg'
+      status: 'PVTSSF_lADOAH2Fnc4ARzsqzgLXyI4'
+      responseDue: 'PVTF_lADOAH2Fnc4ARzsqzgLXyhE'
+  repos:
+    withoutRouting:
```

Comment:
note: once there are repos with routing there may be some issues

Reply:
Say more?

Reply:
Oh! Maybe this isn't what you mean, but we need to figure out Slack channel routing for notifications. 🤔

Back to #482 for that ...

Reply:
Yep, that's what I meant here. We will need to configure a channel to send notifications for support, unless we want to also use the `#discuss-support-open-source` for codecov too.